frontend/src/pages/application/PipelineStage/form.vue

```diff
@@ -448,6 +451,9 @@ export default {
             } else if (this.input.stageType === StageType.DEVICE) {
                 options.unshift({ value: StageAction.USE_ACTIVE_SNAPSHOT, label: 'Use active snapshot' })
             }
+            if (this.isLastStage) {
```

Comment:
```suggestion
            if (!this.isFirstStage && this.isLastStage) {
```

Prevent first stage selecting "none"